vivo-project/VIVO
# Do not offer UMLS vocabulary in interface if umls.properties is absent.

**[https://jira.lyrasis.org/browse/VIVO-1992](https://jira.lyrasis.org/browse/VIVO-1992)**:

Does not include UMLS in the list of vocabulary sources if umls.properties is not present on the resource path.

# How should this be tested?
1. On a person profile, click Rolodex icon next to "research areas".
Click "Add Concept".
List should include AGROVOC, GEMET, LCSH but not UMLS.
2. Leave Tomcat running and add a file called "umls.properties" to WEB-INF/classes/ inside the webapp in Tomcat.
3. Refresh the add concept page.  UMLS should now be offered.

# Interested parties
@VIVO-project/vivo-committers
